docs/visual-basic/reference/command-line-compiler/deterministic.md

```diff
@@ -6,7 +6,7 @@ helpviewer_keywords:
   - "-deterministic compiler option [Visual Basic]"
   - "-deterministic compiler option [Visual Basic]"
 ---
-# -deterministic
+# -deterministic compiler option (Visual Basic)
```

Comment:
Do you want me to add this suffix to all VB compiler options for consistency?

Reply:
yes